ubuntu.22.04/README.md

```diff
@@ -0,0 +1,59 @@
+# Ubuntu WorkerTools
+
+> Please note that we update this document periodically to match the latest version on DockerHub which is publicly available.
+> This does not necessarily match the content of Dockerfiles in this repository, as they may contain changes that are not released yet.
+
+## Image Name
+`octopusdeploy/worker-tools`
+
+## Tags
+
+- `3.3.2-ubuntu.22.04`

```

Comment:
We won't have these tags built for the new ubuntu 22.04 right?

I would've thought only `4.0.0-ubuntu.22.04` (or w/e the next tag) would be available.